backend/src/api/model/search/mod.rs

```diff
@@ -220,6 +240,14 @@ pub(crate) async fn perform(
     Ok(SearchOutcome::Results(SearchResults { items }))
 }
 
+fn is_valid_opencast_uuid(query: &str) -> bool {
```

Comment:
I know this was the name I initially suggested but thinking about it more, `looks_like_opencast_uuid` is a better name I think. This makes it clear that this is just a rough heuristic to trigger our special search logic and not an exact check.